### ISSUE TITLE
Chart legend fails WCAG AA compliance due to bad contrast ratio.

### ISSUE THREAD

**Issue by MiniOranges** [MiniOranges/starwatch#44]
![image](https://user-images.githubusercontent.com/56451580/121563111-bbc3e680-c9df-11eb-8b0b-ea7a3b0c80af.png)

Color appears to be ``rgb(83,83,83)`` on a background of ``rgb(43,43,43)``
